lib/ansible/modules/utilities/logic/include_vars.py

```diff
@@ -44,6 +44,22 @@
       - If omitted (null) they will be made top level vars.
     type: str
     version_added: "2.2"
+  name_format:
+    description:
+      - The format to assign into the variable in I(name).
+      - Ignored if I(name) is omitted.
+      - Defaults to C(dict) which preserves the previous behavior of I(name).
+      - With C(dict_by_path), the variable in I(name) is a C(dict) where each key is the full path of the file
+        from which the vars were loaded, and the value is a C(dict) containing those vars.
+      - C(dict_by_file) is the same as above, but only the C(basename) of the file is used as the key. This means
+        that if a file with the same name exists at a different directory level, the last read file will "win"
+        and overwrite that key's values.
+      - C(list_by_path) and C(list_by_file) work the same way as their C(dict_by_*) counterparts, except that they
+        only return the values of the resulting C(dict), so the variable in I(name) will be a list of those values.
+    default: "dict"
+    choices: [ dict, dict_by_path, dict_by_file, list_by_path, list_by_file ]
+    type: str
+    version_added: "2.10"
```

Comment:
The issue and PR automatically added the [affects_2.9](https://github.com/ansible/ansible/labels/affects_2.9) label, but the tests failed if I didn't change this from 2.9 to 2.10. 

It's not clear to me how to determine the right value for this.